src/documentation/about/tao.malloynb

```diff
@@ -29,13 +29,13 @@ Malloy comprehends data as a network of relationships and so computations like a
 Maybe AppleScript or COBOL would be at the extreme end of this. More of a kind of math, less like a natural language.
 
 ### … is curated
-There are not four different ways to express things depending on which programming language or style you came from. There is one, and it is carefully chosen to match the task of data transformation, discovery and presentation.
+There are not four equal and different ways to express something. In malloy there should be one preferred way that has been carefully chosen to match the task of data transformation, discovery and presentation.
 
 ### … is helpful
-Malloy tries to "do the right thing" that most people want, by default, while still allowing non default expressions to be written. The treatment of `null` and booleans, and the sorting rules for "reduce" stages would be two examples of this.
+Malloy tries to "do the right thing" that most people want, by default, while still allowing non default expressions to be written. The treatment of `null` and booleans, and the sorting rules for "reduce" stages are two examples of this.
 
 ### … is still an experiment
 We had some theoretical insights that there was a better way to interact with data than SQL, and Malloy is the current snapshot of that thinking, but we are not done. We have a number of features which are not yet in the language, which we expect to have an impact on the language, and maybe even on these rules. The language is still young and needs room to grow.
 
 ### … is an expression of empathy towards explorers and explainers of data
-The Malloy user is not someone who writes one sentence in Malloy, and then never sees the language again. Malloy is an invitation into a "way" for people who are passionate about decision making based on data, and good decision making is iterative, and ongoing.
+The Malloy user is not someone who writes one sentence in Malloy, and then never sees the language again. Malloy is an invitation into a "way" (or "Tao") for people who are passionate about decision making based on data; and good decision making is iterative and ongoing.
```

Comment:
"The Tao" literally translated can mean "The way" so there is a clever pun here :D

https://en.wikipedia.org/wiki/Tao

> [Tao] means a road, path, way;